default/scripting/species/common/telepathic.macros

```diff
@@ -0,0 +1,16 @@
+TELEPATHIC_DETECTION
+'''     EffectsGroup
+            description = "TELEPATHIC_DETECTION"
+            scope = And [
+                PopulationCenter
+                Not OR [
+                    Species name = Source.Species
+                    Species name = "SP_EXOBOT"
```

Comment:
If other Trith are immune, should other telepaths be immune as well?
Could exclude by TELEPATHIC tag and maybe TELEPATHIC_IMMUNE for Exobot if you foresee other species having that trait later.